Pull in r198567 from upstream llvm trunk (by Benjamin Kramer):

  SPARC: Make helper function static.

Introduced here: http://svnweb.freebsd.org/changeset/base/262261

Index: lib/Target/Sparc/SparcAsmPrinter.cpp
===================================================================
--- lib/Target/Sparc/SparcAsmPrinter.cpp
+++ lib/Target/Sparc/SparcAsmPrinter.cpp
@@ -130,8 +130,8 @@ static void EmitOR(MCStreamer &OutStreamer, MCOper
   OutStreamer.EmitInstruction(ORInst);
 }
 
-void EmitADD(MCStreamer &OutStreamer,
-             MCOperand &RS1, MCOperand &RS2, MCOperand &RD)
+static void EmitADD(MCStreamer &OutStreamer,
+                    MCOperand &RS1, MCOperand &RS2, MCOperand &RD)
 {
   MCInst ADDInst;
   ADDInst.setOpcode(SP::ADDrr);
